programs/nft-voter/src/state/registrar.rs

```diff
@@ -0,0 +1,97 @@
+use crate::{
+    error::NftVoterError,
+    id,
+    state::CollectionConfig,
+    tools::{spl_token::get_spl_token_amount, token_metadata::get_token_metadata_for_mint},
+};
+use anchor_lang::prelude::*;
+use spl_governance::tools::spl_token::{get_spl_token_mint, get_spl_token_owner};
+
+/// Registrar which stores NFT voting configuration for the given Realm
+#[account]
+#[derive(Debug, PartialEq)]
+pub struct Registrar {
+    /// spl-governance program the Realm belongs to
+    pub governance_program_id: Pubkey,
+
+    /// Realm of the Registrar
+    pub realm: Pubkey,
```

Comment:
nitpick: for direct access you could have the realm_authority directly on the registrar

Reply:
That would be incorrect because the authority stored on `Registrar` could go out of sync with `Realm.authority`   and we should have a single source of truth for it.
This is something we should actually change in `voter-stake-registry`

Reply:
Good find, we should indeed fix vsr